modules/ROOT/pages/configuration-schema-import-filter.adoc

```diff
@@ -9,6 +9,70 @@ include::partial$_set_page_context.adoc[]
 
 include::partial$block-caveats.adoc[tags="legacy-config-equivalents"]
 
+== Introduction
+
+Using an import filter will significantly improve efficiency if you're working with large datasets:
+
+* Import filters are defined at the collection level and allow you to retrieve only the relevant documents you need rather than the entire dataset. 
+The import filter helps determine which documents can be copied by Sync Gateway. 
+It looks at the application's needs and applies these criteria to all future changes.
+
+* By reducing the amount of data that needs to be processed, an import filter will improve the performance of your queries and analysis. 
+It's worth noting that Sync Gateway imports all documents by default, so it's generally a good idea to use an import filter unless you have a strong reason not to.
+
+
+== Function Provision
+
+Use the {rest-api-admin--xref--database-configuration} Admin Rest API endpoint xref:rest_api_admin_static.adoc#get_keyspace-_config-import_filter[/{keyspace}/_config/import_filter] to provision an import filter for a database using the `application/javascript` mime type.
+
+If you are using legacy configuration then, you need to include it in your configuration file, see: {configuration-properties-legacy--xref--databases-import-filter}.
+
+
+== Configuration
+
+
+[#ex-config]
+.Using an Import Filter
+====
+
+[{tabs}]
+=====
+
+API::
++
+--
+
+[source,bash]
+----
+include::{example-cfg}[tag="import-filter-config"]
+----
+
+<.> You will need to provide authentication in the call; unless it is disabled (not recommended in production environment).
+
+--
+
++
+Legacy::
++
+--
+[source,json]
+----
+include::{example-cfg}[tag="import-filter-config-legacy"]
+----
+
+Configuration properties:
+
+<1> The user's username that you created on the Couchbase Server Admin Console.
```

Comment:
Not critical right now, but remember that our style guides prefer to avoid callouts, so if you're already doing a cleanup, can sometimes be a good idea to either do a driveby change or to ticket it for later.

Reply:
I'll remove